profiles/util.py

```diff
@@ -2,11 +2,17 @@
 util functions for profiles
 """
 from datetime import datetime
+from io import BytesIO
 from os import path
 
+from PIL import Image
 import pytz
 
-IMAGE_FILENAME_MAX_LENGTH = 64
+# This is the Django ImageField max path size
+IMAGE_PATH_MAX_LENGTH = 100
+
+# Max dimension of either height or width for a small image
+IMAGE_SMALL_MAX_DIMENSION = 64
```

Comment:
I talked with @roberthouse54 and he doesn't see an immediate need for this to be configurable